content/ja/docs/concepts/policy/resource-quotas.md

```diff
@@ -25,6 +25,11 @@ weight: 10
 - リソースの作成や更新がクォータの制約に違反しているとき、そのリクエストはHTTPステータスコード`403 FORBIDDEN`で失敗し、違反した制約を説明するメッセージが表示されます。
 - `cpu`や`memory`といったコンピューターリソースに対するクォータが名前空間内で有効になっているとき、ユーザーはそれらの値に対する`requests`や`limits`を設定する必要があります。設定しないとクォータシステムがPodの作成を拒否します。 ヒント: コンピュートリソースの要求を設定しないPodに対してデフォルト値を強制するために、`LimitRanger`アドミッションコントローラーを使用してください。この問題を解決する例は[walkthrough](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)で参照できます。
 
+{{< note >}}
+- `cpu`および`memory`リソースに関しては、ResourceQuotaはlimitを設定しているnamespace内の**全ての**（新しい）Podに対して当該リソースのlimitの設定を強制します。namespace内の`cpu`または`memory`どちらかに対してリソースクォータを適用する場合、ユーザやクライアントは全ての新しいPodごとに、そのリソースの`requests`あるいは`limits`を指定**しなければなりません** そうしないと、コントロールプレーンがそのPodの作成を拒否する可能性があります。
+- その他のリソースに関して: ResourceQuotaはリソースのlimitまたはrequestを設定していないpodの場合でも機能します。これは、リソースクォータがnamespace内のエフェメラルストレージに対してlimitを設定している場合、エフェメラルストレージのlimit/requestsを設定していないpodを新規作成できることを意味します。[LimitRange](/docs/concepts/policy/limit-range/)を使うことで、リソースに対して自動的にデフォルト要求を設定することができます。
```

Comment:
```suggestion
- その他のリソースに関して: ResourceQuotaはリソースのlimitまたはrequestを設定していないPodの場合でも機能します。これは、リソースクォータがnamespace内のエフェメラルストレージに対してlimitを設定している場合、エフェメラルストレージのlimit/requestsを設定していないPodを新規作成できることを意味します。[LimitRange](/ja/docs/concepts/policy/limit-range/)を使うことで、リソースに対して自動的にデフォルト要求を設定することができます。
```
- We have the `ja` content of this page, so link to that.
- We should use `Pod` not `pod` I think.

Reply:
done